src/radiation/moments.cpp

```diff
@@ -514,31 +556,92 @@ TaskStatus CalculateFluxesImpl(T *rc) {
           face = CellLocation::Face3;
           break;
         }
+        Real X[4];
+        X[1] = (face == CellLocation::Face1 ? coords.x1f(k, j, i) : coords.x1v(k, j, i));
+        X[2] = (face == CellLocation::Face2 ? coords.x2f(k, j, i) : coords.x2v(k, j, i));
+        X[3] = (face == CellLocation::Face3 ? coords.x3f(k, j, i) : coords.x3v(k, j, i));
+
+        Real W_ceiling, garbage;
```

Comment:
lol `garbage`. Maybe make the final argument optional?

Reply:
Just following the style from `con2prim_robust.hpp` :) Optional final argument might be a bit too specific to satisfy principle-of-least-astonishment for `GetFloors`, and also maybe we will add another floor in the future

Reply:
yeah ok